data/AppEntryClear.css

```
@@ -0,0 +1,20 @@
+/*
+* SPDX-License-Identifier: GPL-2.0-or-later
+* SPDX-FileCopyrightText: 2017-2024 elementary, Inc. (https://elementary.io)
+*/
+
+@keyframes sweep {
+    0%   { -gtk-icon-transform: rotate(0deg);   }
+    25%  { -gtk-icon-transform: rotate(30deg); }
+    50%  { -gtk-icon-transform: rotate(0deg);   }
+    75%  { -gtk-icon-transform: rotate(-30deg);  }
+    100% { -gtk-icon-transform: rotate(0deg);   }
+}
+
+.sweep-animation {
+    transition: all 175ms ease-in-out;
+}
+
+.sweep-animation.active {
+    animation: sweep 700ms ease-in-out;
```

Maybe this should be a little faster to match closer to the timeout?
```suggestion
    animation: sweep 600ms ease-in-out;
```